fe2-android/app/src/main/res/layout/tab_home.xml

```diff
@@ -17,7 +17,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
-    app:layout_constraintGuide_percent="@dimen/guideline_vertical_tab_home" />
+    app:layout_constraintGuide_percent="0.16" />
```

Comment:
It is a good practice to use placeholders for this kind of buttons.

It allows that magic number to be reused and the dimen file can be easily swapped depending on the device model.
(We do not do such a thing, but it might be a thing to consider in the future)